### ISSUE TITLE
Import v8 with go

### ISSUE THREAD

**Issue by FunTimeCoding** [mattermost/mattermost-app-examples#21]
The go example uses v6, and I'm trying to update my go program which uses `github.com/mattermost/mattermost-server` similarly.

Could you update the example on how to use the latest version?

**Comment by hanzei** [comment]
The plugin should work aginst `v8` servers. Did you run into any issues?

**Comment by FunTimeCoding** [comment]
I mean they do. But version 6 isn't maintained anymore, right?

**Comment by hanzei** [comment]
Are you referring to the webhook example in https://github.com/mattermost/mattermost-app-examples/blob/d5f73cd611cb7ec71eb147ef7d8a80e47e27b572/golang/webhooks/go.mod#L7? That does indeed import v6 code, but the code hasn't changed since then. It's fine to still import it, as only `model.Post` is used.

**Comment by FunTimeCoding** [comment]
Yes. I haven't gone digging yet in the v7 or v8 code where this may have gone, but it seems to be structured differently and also doesn't like being imported since the move to the server/ subdirectory.

**Comment by hanzei** [comment]
Surely, it would be nice if the code would import https://pkg.go.dev/github.com/mattermost/mattermost/server/public.

I'm making it a Help Wanted ticket for anyone who wants to jump on it. :rocket:
